client/src/components/MobileMenuToggle/MobileMenuToggle.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Component } from 'react';
+
+class MobileMenuToggle extends Component {
+
+  constructor() {
+    super();
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    e.preventDefault();
+
+    if (typeof this.props.onClick === 'function') {
+      this.props.onClick(e);
+    }
+  }
+
+  render() {
+    const classes = ['cms-mobile-menu-toggle'];
+    if (this.props.isOpen) {
+      classes.push('cms-mobile-menu-toggle--open');
+    }
+
+    return (
+      <button
+        className={classes.join(' ')}
+        href="#toggle-mobile-menu"
+        onClick={this.handleClick}
+        aria-controls={this.props.controls}
+        aria-expanded={!!this.props.isOpen}
+      >
+        <span></span>
+        <span></span>
+        <span></span>
+        <span></span>
+      </button>
+    );
+  }
+}
+
+MobileMenuToggle.propTypes = {
+  isOpen: React.PropTypes.bool,
```

Comment:
I'd suggest making this required, and setting a default value of `false` in `defaultProps`. Right now, `isOpen` is a tri-state prop, taking `true`, `false`, and `undefined`.